runtime/compiler/ras/HashTable.hpp

```diff
@@ -55,6 +55,7 @@ class TR_HashTableEntry
 
    void *operator new[](size_t size, TR_Memory *m);
    void *operator new(size_t size, void *cell) {return cell;}
+   void operator delete[] (void *p, TR_Memory *m) {}
```

Comment:
Kind of surprised that there isn't a warning for the unmatched `delete` for ` void *operator new(size_t size, void *cell) {return cell;}` :S

Reply:
Do you have an example of where it is used? Windows was quite chatty about these missing `delete` operators.

Reply:
Ah I guess it's never used; only the `void *operator new[](size_t size, TR_Memory *m);` is used. That might explain the lack of a warning.

Reply:
nvm, there is a use case:
https://github.com/eclipse-openj9/openj9/blob/7d6dac5f9d54b779d2ba3b5394bf082f2cf0fb83/runtime/compiler/ras/HashTable.cpp#L86

Reply:
It may be that the placement delete that goes with `void *operator new(size_t , void *)` is the standard `operator delete(void *)` so you don't need an explicit `delete`? But I see that you did have an explicit `delete` for other `operator new` with the same signature in other data structures. Did you get warnings for those? Or did you just pre-emptively add the `delete` operators for those?

Reply:
> Did you get warnings for those? Or did you just pre-emptively add the delete operators for those?

I only fixed warnings that were called out by the compiler. Nothing more, nothing less.

Reply:
🤷🏾‍♂️